admin_actions/admin.py

```diff
@@ -293,6 +293,7 @@ def _confirmation_view(
         form_valid_callback,
         form_cls=forms.Form,
         html_template_path="admin/intermediate_action.html",
+        html_context: Optional[dict] = {}
```

Comment:
Optional means that it can be `None`, can it really be?
actual usage
```
**html_context
```
seems not